<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> publications | Matthew Crown </title> <meta name="author" content="Matthew Crown"> <meta name="description" content="A simple, whitespace theme for academics. Based on [*folio](https://github.com/bogoli/-folio) design. "> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link defer rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons&amp;display=swap"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?591dab5a4e56573bf4ef7fd332894c99" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;%F0%9F%A7%AC&lt;/text&gt;&lt;/svg&gt;"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://m-crown.github.io/publications/"> <script src="/assets/js/theme.js?9a0c749ec5240d9cda97bc72359a72c0"></script> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?5847e5ed4a4568527aa6cfab446049ca" media="none" id="highlight_theme_dark"> <script>initTheme();</script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"> <span class="font-weight-bold">Matthew</span> Crown </a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">about </a> </li> <li class="nav-item "> <a class="nav-link" href="/cv/">cv </a> </li> <li class="nav-item active"> <a class="nav-link" href="/publications/">publications <span class="sr-only">(current)</span> </a> </li> <li class="nav-item "> <a class="nav-link" href="/repositories/">repositories </a> </li> <li class="nav-item "> <a class="nav-link" href="/blog/">blog </a> </li> <li class="nav-item"> <button id="search-toggle" title="Search" onclick="openSearchModal()"> <span class="nav-link">ctrl k <i class="ti ti-search"></i></span> </button> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="ti ti-sun-moon" id="light-toggle-system"></i> <i class="ti ti-moon-filled" id="light-toggle-dark"></i> <i class="ti ti-sun-filled" id="light-toggle-light"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5" role="main"> <div class="post"> <header class="post-header"> <h1 class="post-title">publications</h1> <p class="post-description"></p> </header> <article> <p>Publications I have authored or co-authored. I have also been involved in many publications through my work in the COVID-19 Genomics UK Consortium (COG-UK). For a full list of publications, please see my <a href="https://scholar.google.com.hk/citations?user=b8OKEYcAAAAJ" rel="external nofollow noopener" target="_blank">Google Scholar</a> profile.</p> <script src="/assets/js/bibsearch.js?1bc438ca9037884cc579601c09afd847" type="module"></script> <p><input type="text" id="bibsearch" spellcheck="false" autocomplete="off" class="search bibsearch-form-input" placeholder="Type to filter"></p> <div class="publications"> <h2 class="bibliography">2024</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Crown2024-cw" class="col-sm-8"> <div class="title">ProCogGraph: A graph-based mapping of cognate ligand domain interactions</div> <div class="author"> Matthew Crown, and Matthew Bashton </div> <div class="periodical"> <em>Bioinform. Adv.</em>, Oct 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://academic.oup.com/bioinformaticsadvances/advance-article/doi/10.1093/bioadv/vbae161/7831455" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> <div class="abstract hidden"> <p>Abstract Motivation Mappings of domain-cognate ligand interactions can enhance our understanding of the core concepts of evolution and be used to aid docking and protein design. Since the last available cognate-ligand domain database was released, the PDB has grown significantly and new tools are available for measuring similarity and determining contacts. Results We present ProCogGraph, a graph database of cognate-ligand domain mappings in PDB structures. Building upon the work of the predecessor database, PROCOGNATE, we use data-driven approaches to develop thresholds and interaction modes. We explore new aspects of domain-cognate ligand interactions, including the chemical similarity of bound cognate ligands and how domain combinations influence cognate ligand binding. Finally, we use the graph to add specificity to partial EC IDs, showing that ProCogGraph can complete partial annotations systematically through assigned cognate ligands. Availability and Implementation The ProCogGraph pipeline, database and flat files are available at https://github.com/bashton-lab/ProCogGraph and https://doi.org/10.5281/zenodo.13165851. Supplementary information Supplementary data are available at Bioinformatics Advances online.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Gonzalez-Orozco2024-eh" class="col-sm-8"> <div class="title">TRIM7 ubiquitinates SARS-CoV-2 membrane protein to limit apoptosis and viral replication</div> <div class="author"> Maria Gonzalez-Orozco, Hsiang-Chi Tseng, Adam Hage, and <span class="more-authors" title="click to view 22 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '22 more authors' ? 'Hongjie Xia, Padmanava Behera, Kazi Afreen, Yoatzin Peñaflor-Tellez, Maria I Giraldo, Matthew Huante, Lucinda Puebla-Clark, Sarah Tol, Abby Odle, Matthew Crown, Natalia Teruel, Thomas R Shelite, Vineet Menachery, Mark Endsley, Janice J Endsley, Rafael J Najmanovich, Matthew Bashton, Robin Stephens, Pei-Yong Shi, Xuping Xie, Alexander N Freiberg, Ricardo Rajsbaum' : '22 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">22 more authors</span> </div> <div class="periodical"> <em>bioRxivorg</em>, Jun 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://www.biorxiv.org/content/10.1101/2024.06.17.599107v1.abstract" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> <div class="abstract hidden"> <p>SARS-CoV-2 is a highly transmissible virus that causes COVID-19 disease. Mechanisms of viral pathogenesis include excessive inflammation and viral-induced cell death, resulting in tissue damage. We identified the host E3-ubiquitin ligase TRIM7 as an inhibitor of apoptosis and SARS-CoV-2 replication via ubiquitination of the viral membrane (M) protein. Trim7 -/- mice exhibited increased pathology and virus titers associated with epithelial apoptosis and dysregulated immune responses. Mechanistically, TRIM7 ubiquitinates M on K14, which protects cells from cell death. Longitudinal SARS-CoV-2 sequence analysis from infected patients revealed that mutations on M-K14 appeared in circulating variants during the pandemic. The relevance of these mutations was tested in a mouse model. A recombinant M-K14/K15R virus showed reduced viral replication, consistent with the role of K15 in virus assembly, and increased levels of apoptosis associated with the loss of ubiquitination on K14. TRIM7 antiviral activity requires caspase-6 inhibition, linking apoptosis with viral replication and pathology.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="McKay2024-oz" class="col-sm-8"> <div class="title">Environmental microbiome in the home and daycare settings during the COVID-19 pandemic, and potential risk of non-communicable disease in children</div> <div class="author"> Jill A McKay, Matthew Crown, Matthew Bashton, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'David Pearce, Jane A Entwistle, Vartul Sangal' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>Environ. Microbiol. Rep.</em>, Feb 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10866607/" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> <div class="abstract hidden"> <p>An exposure to diverse microbial population early in life is important for the development of immunity against various non-communicable diseases including asthma, childhood leukaemia and other cancers. Social mixing in daycare settings helps with exposure to a variety of microbes. However, social isolation and a high emphasis on workplace hygiene during the COVID pandemic may have affected children’s exposure to diverse microbiota. The structure of microbial communities and their role in developing immunity to various diseases are not well understood. In this study, we investigated the structure of microbial communities in daycare and home settings during the pandemic. Interestingly, microbial diversity was relatively higher in dust samples collected from homes, with human-associated taxa being more prevalent compared to those from daycare settings. Environmental microbes were more abundant in dust samples from daycare providers. These results potentially suggest that cleaning practices during the pandemic may have influenced the diversity and microbial abundance of the daycare samples. Several bacterial taxa detected in both the environments are known to induce anti-inflammatory and immunomodulatory responses, conferring protection from various diseases. Therefore, exposure to diverse microbial population in early childhood may play an important role in developing immunity against various non-communicable and infectious diseases.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Crown2024-cz" class="col-sm-8"> <div class="title">OMEinfo: global geographic metadata for -omics experiments</div> <div class="author"> Matthew Crown, and Matthew Bashton </div> <div class="periodical"> <em>Bioinform. Adv.</em>, Feb 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="http://dx.doi.org/10.1093/bioadv/vbae025" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> <div class="abstract hidden"> <p>Summary: Microbiome studies increasingly associate geographical features like rurality and climate with microbiomes. It is essential to correctly integrate rich geographical metadata; and inconsistent definitions of rurality, can hinder cross-study comparisons. We address this with OMEinfo, a tool for automated retrieval of consistent geographical metadata from user-provided location data. OMEinfo leverages open data sources such as the Global Human Settlement Layer, and Open-Data Inventory for Anthropogenic Carbon dioxide. OMEinfo’s web-app enables users to visualize and investigate the spatial distribution of metadata features. OMEinfo promotes reproducibility and consistency in microbiome metadata through a standardized metadata retrieval approach. To demonstrate utility, OMEinfo is used to replicate the results of a previous study linking population density to bacterial diversity. As the field explores relationships between microbiomes and geographical features, tools like OMEinfo will prove vital in developing a robust, accurate, and interconnected understanding of these interactions, whilst having applicability beyond this field to any studies utilizing location-based metadata. Finally, we release the OMEinfo annotation dataset of 5.3 million OMEinfo annotated samples from the ENA, for use in retrospective analyses of sequencing samples, and suggest several ways researchers and sequencing read repositories can improve the quality of underlying metadata submitted to these public stores. Availability and implementation: OMEinfo is freely available and released under an MIT licence. OMEinfo source code is available at https://github.com/m-crown/OMEinfo/ and https://doi.org/10.5281/zenodo.10518763.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2022</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Crown2022-qa" class="col-sm-8"> <div class="title">SPEAR: Systematic ProtEin AnnotatoR</div> <div class="author"> Matthew Crown, Natália Teruel, Rafael Najmanovich, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Matthew Bashton' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>Bioinformatics</em>, Aug 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="http://dx.doi.org/10.1093/bioinformatics/btac391" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> <div class="abstract hidden"> <p>SUMMARY: We present Systematic ProtEin AnnotatoR (SPEAR), a lightweight and rapid SARS-CoV-2 variant annotation and scoring tool, for identifying mutations contributing to potential immune escape and transmissibility (ACE2 binding) at point of sequencing. SPEAR can be used in the field to evaluate genomic surveillance results in real time and features a powerful interactive data visualization report. AVAILABILITY AND IMPLEMENTATION: SPEAR and documentation are freely available on GitHub: https://github.com/m-crown/SPEAR and are implemented in Python and installable via Conda environment. SUPPLEMENTARY INFORMATION: Supplementary data are available at Bioinformatics online.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2021</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Thompson2021-jg" class="col-sm-8"> <div class="title">Bacterial Diversity in House Dust: Characterization of a Core Indoor Microbiome</div> <div class="author"> Jonathan R Thompson, Ariadne Argyraki, Matthew Bashton, and <span class="more-authors" title="click to view 12 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '12 more authors' ? 'Lindsay Bramwell, Matthew Crown, Andrew S Hursthouse, Khadija Jabeen, Paula Marinho Reis, Anil Namdeo, Andrew Nelson, David A Pearce, Sanja Potgieter-Vermaak, Pat E Rasmussen, Joanna Wragg, Jane A Entwistle' : '12 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">12 more authors</span> </div> <div class="periodical"> <em>Front. Environ. Sci. Eng. China</em>, Aug 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://www.frontiersin.org/articles/10.3389/fenvs.2021.754657" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> <div class="abstract hidden"> <p>Our indoor microbiome consists of a wide range of microbial taxa. Whilst many of these microbes are benign, some are beneficial, some harmful, yet our knowledge of the spatial heterogeneity of bacterial assemblages in our residential environment remains limited. To investigate the existence of a common core house dust bacterial microbiome we selected household vacuum dusts, collected through a citizen science approach, from homes across two bioclimatic regions (UK, Oceanic/Maritime and Greece, Mediterranean). Following the extraction of DNA from each dust sample, we targeted the bacterial 16S rRNA gene using Illumina NextSeq sequencing. PERMANOVA analysis of the microbial communities at family level grouped samples within their distinct bioclimatic region and SIMPER analysis at genus level identified the statistically significant taxa responsible for driving diversity between these groups. A “common to all” core house dust microbiome consisted of Acinetobacter, Massalia, Rubellimicrobium, Sphingomonas and Staphylococcus; genera typically associated with human occupancy and common environmental sources. Additionally, a “unique location specific” microbiome was identified, reflective of the bioclimatic region. The Greek dusts indicated a lower average diversity than the UK house dusts, with a high abundance of Rhizobiaceae in the Greek samples. Our study highlights citizen science as a powerful approach to access the indoor residential environment, at scale, and establishes the existence of a “core” house dust microbiome independent of bioclimatic region.</p> </div> </div> </div> </li></ol> </div> </article> </div> </div> <footer class="fixed-bottom" role="contentinfo"> <div class="container mt-0"> © Copyright 2025 Matthew Crown. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. Photos from <a href="https://unsplash.com" target="_blank" rel="external nofollow noopener">Unsplash</a>. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@5.0.0/imagesloaded.pkgd.min.js" integrity="sha256-htrLFfZJ6v5udOG+3kNLINIKh2gvoKqwEhHYfTTMICc=" crossorigin="anonymous"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.1.0/dist/medium-zoom.min.js" integrity="sha256-ZgMyDAIYDYGxbcpJcfUnYwNevG/xi9OHKaR/8GK+jWc=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?85ddb88934d28b74e78031fd54cf8308"></script> <script src="/assets/js/no_defer.js?2781658a0a2b13ed609542042a859126"></script> <script defer src="/assets/js/common.js?e0514a05c5c95ac1a93a8dfd5249b92e"></script> <script defer src="/assets/js/copy_code.js?12775fdf7f95e901d7119054556e495f" type="text/javascript"></script> <script defer src="/assets/js/jupyter_new_tab.js?d9f17b6adc2311cbabd747f4538bb15f"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.2/es5/tex-mml-chtml.js" integrity="sha256-MASABpB4tYktI2Oitl4t+78w/lyA+D7b/s9GEP0JOGI=" crossorigin="anonymous"></script> <script defer src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6" crossorigin="anonymous"></script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> <script src="/assets/js/vanilla-back-to-top.min.js?f40d453793ff4f64e238e420181a1d17"></script> <script>addBackToTop();</script> <script type="module" src="/assets/js/search/ninja-keys.min.js?a3446f084dcaecc5f75aa1757d087dcf"></script> <ninja-keys hidebreadcrumbs noautoloadmdicons placeholder="Type to start searching"></ninja-keys> <script>let searchTheme=determineComputedTheme();const ninjaKeys=document.querySelector("ninja-keys");"dark"===searchTheme?ninjaKeys.classList.add("dark"):ninjaKeys.classList.remove("dark");const openSearchModal=()=>{const e=$("#navbarNav");e.hasClass("show")&&e.collapse("hide"),ninjaKeys.open()};</script> <script>const ninja=document.querySelector("ninja-keys");ninja.data=[{id:"nav-about",title:"about",section:"Navigation",handler:()=>{window.location.href="/"}},{id:"nav-cv",title:"cv",description:"",section:"Navigation",handler:()=>{window.location.href="/cv/"}},{id:"nav-publications",title:"publications",description:"",section:"Navigation",handler:()=>{window.location.href="/publications/"}},{id:"nav-repositories",title:"repositories",description:"A collection of projects I have worked on. Check out my GitHub profile for more!",section:"Navigation",handler:()=>{window.location.href="/repositories/"}},{id:"nav-blog",title:"blog",description:"",section:"Navigation",handler:()=>{window.location.href="/blog/"}},{id:"post-an-introduction",title:"An Introduction",description:"Welcome!",section:"Posts",handler:()=>{window.location.href="/blog/2024/introduction/"}},{id:"news-a-simple-inline-announcement",title:"A simple inline announcement.",description:"",section:"News"},{id:"news-a-long-announcement-with-details",title:"A long announcement with details",description:"",section:"News",handler:()=>{window.location.href="/news/announcement_2/"}},{id:"news-a-simple-inline-announcement-with-markdown-emoji-sparkles-smile",title:'A simple inline announcement with Markdown emoji! <img class="emoji" title=":sparkles:" alt=":sparkles:" src="https://github.githubassets.com/images/icons/emoji/unicode/2728.png" height="20" width="20"> <img class="emoji" title=":smile:" alt=":smile:" src="https://github.githubassets.com/images/icons/emoji/unicode/1f604.png" height="20" width="20">',description:"",section:"News"},{id:"projects-project-1",title:"project 1",description:"with background image",section:"Projects",handler:()=>{window.location.href="/projects/1_project/"}},{id:"projects-project-2",title:"project 2",description:"a project with a background image and giscus comments",section:"Projects",handler:()=>{window.location.href="/projects/2_project/"}},{id:"projects-project-3-with-very-long-name",title:"project 3 with very long name",description:"a project that redirects to another website",section:"Projects",handler:()=>{window.location.href="/projects/3_project/"}},{id:"projects-project-4",title:"project 4",description:"another without an image",section:"Projects",handler:()=>{window.location.href="/projects/4_project/"}},{id:"projects-project-5",title:"project 5",description:"a project with a background image",section:"Projects",handler:()=>{window.location.href="/projects/5_project/"}},{id:"projects-project-6",title:"project 6",description:"a project with no image",section:"Projects",handler:()=>{window.location.href="/projects/6_project/"}},{id:"projects-project-7",title:"project 7",description:"with background image",section:"Projects",handler:()=>{window.location.href="/projects/7_project/"}},{id:"projects-project-8",title:"project 8",description:"an other project with a background image and giscus comments",section:"Projects",handler:()=>{window.location.href="/projects/8_project/"}},{id:"projects-project-9",title:"project 9",description:"another project with an image \ud83c\udf89",section:"Projects",handler:()=>{window.location.href="/projects/9_project/"}},{id:"socials-email",title:"Send email",section:"Socials",handler:()=>{window.open("mailto:%6D%61%74%74%68%65%77%63%72%6F%77%6E@%68%6F%74%6D%61%69%6C.%63%6F.%75%6B","_blank")}},{id:"socials-google-scholar",title:"Google Scholar",section:"Socials",handler:()=>{window.open("https://scholar.google.com/citations?user=qc6CJjYAAAAJ","_blank")}},{id:"socials-github",title:"GitHub",section:"Socials",handler:()=>{window.open("https://github.com/m-crown","_blank")}},{id:"socials-rss",title:"RSS Feed",section:"Socials",handler:()=>{window.open("/feed.xml","_blank")}},{id:"light-theme",title:"Change theme to light",description:"Change the theme of the site to Light",section:"Theme",handler:()=>{setThemeSetting("light")}},{id:"dark-theme",title:"Change theme to dark",description:"Change the theme of the site to Dark",section:"Theme",handler:()=>{setThemeSetting("dark")}},{id:"system-theme",title:"Use system default theme",description:"Change the theme of the site to System Default",section:"Theme",handler:()=>{setThemeSetting("system")}}];</script> <script src="/assets/js/shortcut-key.js?6f508d74becd347268a7f822bca7309d"></script> </body> </html>